src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -772,6 +819,33 @@
           "exclude": [
             "MyExtensionsApp/appsettings.*"
           ]
+        },
+        {
+          "condition": "(useAzureDevOps)",
+          "rename": {
+            "build": "azure"
+          },
+          "exclude": [
+            "github/**/*"
+          ]
+        },
+        {
+          "condition": "(useGithubActions)",
+          "rename": {
+            "github": ".github"
```

Comment:
Is there a limitation with templates where we can't just have .github as the folder name? or are we concerned that it might impact the workflows in the extensions repo?

Reply:
i don't remember the exact reason but i discussed this with @dansiegel

Reply:
@nickrandolph this was at my request. We can discuss later